ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -252,7 +252,11 @@ - (dispatch_queue_t)methodQueue {
 }
 
 RCT_EXPORT_METHOD(getUserAttribute:(NSString *)key callback:(RCTResponseSenderBlock)callback) {
+    @try {
     callback(@[[Instabug userAttributeForKey:key]]);
+    } @catch (NSException *exception) {
+        callback(nil);
```

Comment:
I'm not sure about passing `nil` directly, it's stated in the docs that callback accepts an array and inside that array, we pass our params. This is [an example](https://reactnative.dev/docs/native-modules-ios#callbacks) from their docs just to be safe:

```suggestion
        callback(@[[NSNull null]]);
```